apps/common/components/AppHeader.tsx

```diff
@@ -125,7 +131,7 @@ export function	AppHeader(): ReactElement {
 	}, [pathname]);
 
 	const	supportedNetworks = useMemo((): number[] => {
-		if (pathname.startsWith('/ycrv') || pathname.startsWith('/veyfi') || pathname.startsWith('/ybribe')) {
+		if (pathname.startsWith('/ycrv') || pathname.startsWith('/ybal') || pathname.startsWith('/veyfi') || pathname.startsWith('/ybribe')) {
```

Comment:
nit: It's starting to get a bit long, how about

```suggestion
	const mainnetPaths = ['/ycrv', '/ybal', '/veyfi', '/ybribe'];
	if (mainnetPaths.some(path => pathname.startsWith(path))) {
		return [1];
	}
```